i18n/locales/en.ts

```diff
@@ -1343,22 +1343,36 @@ const translations = {
         nav_title: 'The input class',
         heading: 'Looking at the Input class implementation',
         paragraph_one:
-          'Bitcoin transaction inputs always point to existing, unspent transaction outputs. Therefore, our Input class has a method <span className="text-green"> from_output() </span> which is used to construct an Input by passing in the output description:',
+          'Here we have code for two classes: an <span className="font-bold">Input</span> class and an <span className="font-bold">Outpoint</span> class.',
         paragraph_two:
-          'The first two arguments are the transaction ID and the index of the output of that transaction you want to spend from. Eventually we will pass in the txid and vout values you got above from listunspent. ',
-        paragraph_three: {
-          a: 'Hashes in Bitcoin are ',
+          'Inputs come from unspent transaction outputs. If you provide the description of an output to the <span className="text-green p-1 font-mono bg-[#00000033] m-1 text-base">from_output()</span> method, it will create an instance of the Input class:',
+        paragraph_three: 'The first two arguments are:',
+        paragraph_four:
+          '1. <span className="font-bold">txid: </span>the ID of the transaction that created the output, and',
+        paragraph_five:
+          '2. <span className="font-bold">vout: </span>the index of the output in the transaction\'s entire list of outputs',
+        paragraph_six:
+          'Together, these two pieces of information make up an <span className="font-bold">Outpoint</span>. Eventually we will pass in the txid and vout values that came from executing the <span className="text-green p-1 font-mono bg-[#00000033] m-1 text-base">listunspent</span> command in the previous exercise. ',
+        heading_two: 'A note on endianness',
+        paragraph_seven:
+          "The second two arguments are the value of the output we want to spend (in satoshis) and something called a scriptcode. For now, we will just store that data as an empty byte array, we won't need it until later.",
+        paragraph_eight: {
+          a: 'When we see a Bitcoin hash as a hexadecimal string, the data is actually',
           b: {
             text: ' reversed ',
             question: 'Why do we reverse hashes in bitcoin?',
             href: 'https://chat.bitcoinsearch.xyz/?author=holocat&question=why%2520are%2520hashes%2520reversed%2520in%2520bitcoin',
           },
-          c: ' when we see them as hexadecimal strings. When we accept hashes as strings from a user we must reverse the byte order before storing or transmitting them as raw bytes. This is why we reverse the byte order of the txid argument that is passed in here.',
+          c: '. This ordering of bytes is called ',
+          d: {
+            text: 'endianness',
```

Comment:
Only numbers have endianness. It is not correct to say that a hash is big or little endian. Bitcoin block hashes actually come out of the hash function with the 0's on the *right side*. Satoshi decided to interpret that as a little-endian integer. Sometime later when the JSON-RPC was eventually added, the decision was made to display those block hashes backwards so that they looked more like conventional big endian integers.

So what we do to hashes (sipa would say) is "byte swap" or just "reverse" them.

Reply:
Endianness only applies to numbers. Endianness only applies to numbers. Endianness only applies to numbers. 

I need to get a tattoo of this!!!!! Ty for the feedback! To refresh my memory, I also dug up some old comments you had  made that basically reiterate what you said here. 

I'll update this PR to use "byte swap" or "reverse"